src/System.Text.Encoding/tests/EncodingTestHelpers.cs

```diff
@@ -10,6 +11,12 @@ public static class EncodingHelpers
     {
         private static readonly RandomDataGenerator s_randomGenerator = new RandomDataGenerator();
 
+        public static IEnumerable<object[]> Encodings_TestData()
```

Comment:
With this change, you're converting EncodingHelpers into actually having tests.  Is the plan to remove the per-Encoding tests (except for special ones) and augment this TestData with an entry for each encoding?  Should EncodingHelpers be renamed then?